src/components/Table/useReelShadow.ts

```diff
@@ -0,0 +1,28 @@
+import { useTheme } from '../../hooks/useTheme'
+
+type ReelShadow = {
+  showShadow?: boolean
+  direction?: 'left' | 'right'
+}
+
+export const useReelShadow = ({ showShadow = true, direction = 'left' }: ReelShadow) => {
+  const theme = useTheme()
+  const shadowWidth = theme.space(0.75)
+
+  return `
+    &::after {
+      content: '';
+      position: absolute;
+      z-index: 0;
+      inset-block: 0;
+      left: ${direction === 'left' ? '0' : `-${shadowWidth}`};
+      width: ${shadowWidth};
+      pointer-events: none; /* 影の領域が広すぎるとクリッカブルエリアを侵食するので無効化 */
+      background: linear-gradient(${
+        direction === 'left' ? '90deg' : '-90deg'
+      }, rgba(0, 0, 0, 0.2) 0%, rgba(0, 0, 0, 0) 100%);
```

Comment:
透明色は使えなそうですか?
https://smarthr.design/products/design-tokens/color/#h3-6

Reply:
@uknmr 透明色についてはかなり調整したので、このまま行きたい気持ちがありますー。`rgba(0, 0, 0, 0.2)`は頑張れば`TRANSPARENCY_15`に置き換えられそうだけど微妙に変わってしまいます。
`rgba(0, 0, 0, 0)`は`TRANSPARENCY_0`が合っても良さそう（別チケかな）

Reply:
`rgba(0, 0, 0, 0)` は `transparent` でどうですか?

```suggestion
      }, rgba(0, 0, 0, 0.2) 0%, transparent 100%);
```

Reply:
あー、transparentで使えるんですね！（知らなかった）